docs/about.md

```diff
@@ -0,0 +1,73 @@
+# About Heptio Ark
+
+Heptio Ark provides customizable degrees of recovery for all Kubernetes API objects (Pods, Deployments, Jobs, Custom Resource Definitions, etc.), as well as for persistent volumes. This recovery can be cluster-wide, or fine-tuned according to object type, namespace, or labels.
+
+Ark is ideal for the disaster recovery use case, as well as for snapshotting your application state, prior to performing system operations on your cluster (e.g. upgrades).
+
+Ark consists of a server and a client.
+
+## Server
+
+Ark provides the following operations:
+
+* Backups
+* Schedules
+* Restores
+
+Each operation is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. An additional custom resource, Config, specifies cloud provider settings. These resources are handled by [custom controllers][21] when their corresponding requests are submitted to the Kubernetes API server.
+
+Each controller watches its custom resource for API requests (Ark operations), performs validation, and handles most of the logic for interacting with the cloud provider API -- for example, managing object storage and persistent volumes.
```

Comment:
`most of the logic` makes me wonder as a reader what it doesn't handle/why that's not explained

Reply:
Ditto - let's remove the ambiguity. We handle all of it :)